ObjectsComparer/ObjectsComparer/Enums/ComparisonStatus.cs

```diff
@@ -0,0 +1,19 @@
+﻿// ***********************************************************************
+// Assembly         : Objects Comparer
+// Author           : Ankur Kumar Gupta
+// Created          : 24-Feb-2022
+// ***********************************************************************
+
+namespace ObjectsComparer.Enums
+{
+  /// <summary>
+  /// Specifies the status of comparison i.e. whether to include in comparison or not
+  /// </summary>
+  public enum ComparisonStatus
+  {
+    /// <summary>
+    /// Specifies that the element on which it is applied has to be excluded from comparison
+    /// </summary>
+    ExcludeFromComparison
```

Comment:
What is the benefits of having this enum with 1 value?

Reply:
Currently, I have added just one value in the enum as per my requirements, but further, we can extend enums to modify more settings like `IgnoreRecursiveComparison ` - if we want only one item to ignore for recursive comparison. So I am keeping my options open to manage comparison settings in one enum class.